ClearBank.ApiTest/ClearBank.ApiTest/Steps/DeckOfCardsSteps.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ClearBank.ApiTest.Models;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using RestSharp;
+using System;
+using TechTalk.SpecFlow;
+
+namespace ClearBank.ApiTest
+{
+    [Binding]
+    public class DeckOfCardsSteps
+    {
+        private RestClient restClient;
+        private RestRequest restRequest;
+        private String deckId;
+        private String remaining;
+     
+
+        [Given(@"I create a new deck of cards")]
+        public void GivenICreateANewDeckOfCards()
+        {
+            restClient = new RestClient("https://deckofcardsapi.com/api/");
```

Comment:
Personally I think you could have abstracted this code into a lower lever. Similar to a Page object modelling approach when doing GUI automation. 

That way you could directly call a method that does all of these individual lines, thus having a one-liner code call in your step definition. 

The benefit is that you separate the document layer from the code layer as it should be. (Cucumber best practices)